include-in-header.html

```diff
@@ -13,123 +13,145 @@
 
 <!-- Google Tag Manager -->
 <script>
-  (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-KHBDBW7');
+  (function (w, d, s, l, i) {
```

Comment:
I do think applying prettier to the entire file is a bit aggressive in this particular file, especially for these lines.

Can we just prettify selections instead?